composer.json

```diff
@@ -1,13 +1,8 @@
 {
-    "name": "accessibilityreport",
+    "name": "itk-dev/webaccessibilitystatement",
```

Comment:
For consistency with the module name I think this should be `itk-dev/web_accessibility_statement`. And we should probably rename the repository too.

Reply:
Fixed